<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>RePaLk AI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/logo.png</normaloff>../icons/logo.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <property name="statusTip">
   <string comment="Save to a file"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
border: 1.5px solid rgb(111, 111, 0);
color:black</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Press enter to input text to output window </string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
padding:5px;
image-postion 5px,0px
}
QPushButton:hover{
background-color: rgb(208, 0, 104);
}

</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/enter.png</normaloff>../icons/enter.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>50</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>450</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>text input </string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
background-color:rgb(244, 255, 147);
font: italic 12pt &quot;Calibri&quot;;
color:rgb(26, 77, 38);
border-radius:15px;
padding-left:5px
}

QTextEdit:focus{
border: 1.5px solid rgb(129, 0, 97);
}</string>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>0</number>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri'; font-size:12pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>Enter a Prompt</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>760</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Output Window</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0568182 rgba(172, 242, 231, 255), stop:0.869318 rgba(255, 215, 215, 255));
padding:4px; 
border: 2px solid rgb(115, 115, 115);
border-radius:10px;
}

QListWidget::item{
padding:4px;
margin:4px;
border-radius: 7px;
 background-color:rgb(244, 180, 26); 
color:rgb(20, 61, 89);
font: 12pt &quot;Arial Rounded MT Bold&quot;;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::Batched</enum>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>530</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string extracomment="'Save to a file'">Select one output item at a time and click to save into a file</string>
   </property>
   <property name="statusTip">
    <string extracomment="save to a file"/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
}
QPushButton:hover{
background-color: rgb(208, 0, 104);
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/Save2file.png</normaloff>../icons/Save2file.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>50</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Press until Voice command show in Output Window</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
border-radius: 12px;
}

QPushButton:hover
{background-color: rgb(208, 0, 104);}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/microphone.png</normaloff>../icons/microphone.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>60</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>530</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Press until Voice command show in Output Window</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
border-radius: 12px;
color:rgb(43, 0, 0)
}

QPushButton:hover
{background-color: rgb(208, 0, 104);}
</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>60</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icons/enter.qrc"/>
 </resources>
 <connections/>
</ui>
